spec/system/login_spec.rb

```diff
@@ -226,9 +235,9 @@ def wait_for_email_link(user, type)
     end
 
     it "can reset password with a backup code" do
-      login_modal.open
-      login_modal.fill_username("john")
-      login_modal.forgot_password
+      login_form.open
+      login_form.fill_username("john")
+      login_form.forgot_password
```

Comment:
if you return self in each of these helpers, you can rewrite all these specs to:

`login_form.open.fill_username("john").forgot_password`